scss/helpers/_general.scss

```diff
@@ -6,6 +6,12 @@ $ns: ns();
 .#{$ns}text-transform-none {
     text-transform: none !important;
 }
+.#{$ns}has-text-bold {
```

Comment:
These should be `has-font-weight-bold` and `has-font-style-italic`. With a few exceptions, the helpers use the exact CSS declaration terminology.